### PR TITLE
fix bug in project

### PR DESCRIPTION
**Pull request by archit7448**
# What I have done 

1.  Work on token updates and flow in-app 
2. fix a bug in signUpHandler

### CODE REVIEW

**Review by iHarryD — APPROVED** [review]
- Inconsistent naming. Function names should be written in camel case, and not in pascel case unless they are components.
- Reducer functions need to be pure. Instead of accessing token from within localStorage in the reducer itself, pass it in using payload.
- You shouldn't use the document object to set title. You could set the title directly in the index.html file.
- Putting onClick events on elements such h3 is not recommended for accessibility reasons. You could wrap the h3 element inside a button element and then put onClick event on it.

**Comment by archit7448** [comment]
> * Inconsistent naming. Function names should be written in camel case, and not in pascel case unless they are components.
> * Reducer functions need to be pure. Instead of accessing token from within localStorage in the reducer itself, pass it in using payload.
> * You shouldn't use the document object to set title. You could set the title directly in the index.html file.
> * Putting onClick events on elements such h3 is not recommended for accessibility reasons. You could wrap the h3 element inside a button element and then put onClick event on it.


Thanks for the suggestion I will look into that.

File: src/Components/VideoCard/videoCard.jsx [hunk 1]
```diff
@@ -4,14 +4,13 @@ import { BsThreeDotsVertical } from "react-icons/bs";
 import "./videoCard.css";
 import { useNavigate } from "react-router-dom";
 import { useState } from "react";
-import { AddWatchLater } from "../../reducer/watchLater";
-import { AddHistory } from "../../reducer/history";
+import { addWatchLater } from "../../reducer/watchLater";
+import { addHistory } from "../../reducer/history";
```
Comment (bhushannaikaroskar) [inline src/Components/VideoCard/videoCard.jsx]: You can use index.js file in reducer folder and export all the reducers at once this way you don't have to write single line for each import you can import all at once

**Review by bhushannaikaroskar — APPROVED** [review]
Hey archit your code is good. I have provided some suggestion please do check. Also I am curious to know why you changed the token variable from Auth to Data Provider, any specific reason?

**Comment by archit7448** [comment]
> Hey archit your code is good. I have provided some suggestion please do check. Also I am curious to know why you changed the token variable from Auth to Data Provider, any specific reason?

Token variable in data provider , because I have use data provider usually in every component, if I have to use from auth provider then I have to write lines of code again and again.

**Pull request merged**